vite.config.js

```diff
@@ -110,7 +112,7 @@ export default ({ command, mode }) => {
     },
     build: {
       rollupOptions: {
-        plugins: [visualizer()],
+        plugins: rollupPlugins
```

Comment:
`rollupPlugins` is defined in `rollup` and is where the logic is to run different things in each environment?